src/components/button/button.scss

```diff
@@ -0,0 +1,10 @@
+.button {
+  font-family: Titillium Web;
```

Comment:
We won't need this one here. It's been cover in this PR https://github.com/code4romania/taskforce-fe-components/pull/21

Same for the `.storybook/preview-head.html` file
```suggestion
```